README.md

```diff
@@ -93,6 +95,24 @@ jobs:
             with:
                 php-version: 7.3
                 coverage: none # disable xdebug, pcov
+                tools: cs2pr
```

Comment:
you could even install phpstan via the `shivammathur/setup-php` action, but I am ok with leaving this for now.

Reply:
well, i'd advise not, since each time a build will get triggered, phpstan could be using a different version.

So for pull request A, 
commit 1 : phpstan 0.12.1
commit 2; phpstan 0.12.3

Always prefer requiring phpstan, phpcs and php-cs-fixer with dev dependancies, so you can control when to update them, and when to fix new cs errors reported by newest versions

Reply:
I totally agree and dont use phpstan for exactly this arguments

Reply:
And you can even run those tools locally if you added them in your composer.json :D